00-flex.yaml

```
@@ -46,6 +46,10 @@ template: |
             set -x -e
 
             curl -s https://get.symfony.com/cloud/configurator | (>&2 bash)
+            {{ range $ext := php_extensions -}}
+            {{- if not php_extension_available $ext $.PhpVersion -}}
+            (>&2 php-ext-install {{ $ext }} version)
```

I would remove "version" here so that the command works out of the box without any changes (same below).

---

I won't as a version is required

---

Let's comment the command then?

---

you mean like this:
```suggestion
            # (>&2 php-ext-install {{ $ext }} version)
```
?

---

or like this maybe:
```suggestion
            # (>&2 php-ext-install {{ $ext }} X.Y.Z)
```
?

---

Second one